lib/stringRev.rb

```diff
@@ -0,0 +1,4 @@
+puts "enter a string"
+input = gets.chomp
+
+puts input.reverse
```

Comment:
While correct, I believe the prompt was looking for you to reverse the string without using the built-in method